packages/lodestar/test/unit/sync/utils/assertPeerRelevance.test.ts

```diff
@@ -99,4 +99,20 @@ describe("sync / utils / assertPeerRelevance", () => {
       }
     });
   }
+
+  it("Accept during pre-genesis clock", () => {
+    // clock slot pre-genesis (< 0) by a good margin
+    (chain as any).clock.currentSlot = -50;
```

Comment:
Would be nicer to not cast as any (in case we rename or move clock we won't detect error here) but unlikely to happen so I guess it's fine